src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -0,0 +1,111 @@
+        {
+        }
+
+        /// <summary>
+        /// Gets the usage instructions for the code generator.
+        /// </summary>
+        public override string UsageInstructions => @"The ""gem 'ms_rest_azure' ~> 0.5"" is required for working with generated code.";
+
+        /// <summary>
+        /// Generates Ruby code for Azure service client.
+        /// </summary>
+        /// <param name="cm">The code model.</param>
+        /// <returns>Async tasks which generates SDK files.</returns>
+        public override async Task Generate(CodeModel cm)
+        {
+            var codeModel = cm as CodeModelRba;
+            if (codeModel == null)
+            {
+                throw new InvalidCastException("CodeModel is not a Azure Ruby code model.");
+            }
+
+            // Service client
+            var serviceClientTemplate = new ServiceClientTemplate { Model = codeModel };
+            await Write(serviceClientTemplate, Path.Combine(GeneratorSettingsRba.Instance.sdkPath, CodeNamer.UnderscoreCase(codeModel.Name) + ImplementationFileExtension));
+
+            // Operations
+            foreach (MethodGroupRba group in codeModel.Operations.Where(each => !each.IsCodeModelMethodGroup))
+            {
+                // Operation
+                var operationsTemplate = new AzureMethodGroupTemplate { Model = group };
+                await Write(operationsTemplate, Path.Combine(GeneratorSettingsRba.Instance.sdkPath, CodeNamer.UnderscoreCase(operationsTemplate.Model.TypeName) + ImplementationFileExtension));
+            }
+
+            // Models
+            foreach (CompositeTypeRba model in codeModel.ModelTypes)
+            {
+                if ((model.Extensions.ContainsKey(AzureExtensions.ExternalExtension) &&
+                    (bool)model.Extensions[AzureExtensions.ExternalExtension])
+                    || model.Name == "Resource" || model.Name == "SubResource")
+                {
+                    continue;
+                }
+
+                if( codeModel.pageModels.Any( each => each.Name.EqualsIgnoreCase(model.Name ) ) )
+                {
+                    // Skip, handled in the .pageModels section below.
+                    continue;
+                }
+
+                var modelTemplate = new ModelTemplate { Model = model };
+                await Write(modelTemplate, Path.Combine(GeneratorSettingsRb.Instance.modelsPath, CodeNamer.UnderscoreCase(model.Name) + ImplementationFileExtension));
+            }
+            // Paged Models
+            foreach (var pageModel in codeModel.pageModels)
+            {
+                var pageTemplate = new PageModelTemplate { Model = pageModel };
+                await Write(pageTemplate, Path.Combine(GeneratorSettingsRb.Instance.modelsPath, CodeNamer.UnderscoreCase(pageModel.Name) + ImplementationFileExtension));
+            }
+
+            // Enums
+            foreach (var enumType in codeModel.EnumTypes)
+            {
```

Comment:
There is something changes here...for the enum model name it should 

```
    module PolicyType
      NotSpecified = "NotSpecified"
      BuiltIn = "BuiltIn"
      Custom = "Custom"
    end
```
https://github.com/Azure/azure-sdk-for-ruby/blob/master/management/azure_mgmt_policy/lib/generated/azure_mgmt_policy/models/policy_type.rb#L11

but now it is 

```
    module policyType
      NotSpecified = "NotSpecified"
      BuiltIn = "BuiltIn"
      Custom = "Custom"
    end
```

capitalization on module name is different.